modules/osdk-ansible-managing-cr-status.adoc

```diff
@@ -54,17 +54,16 @@ with a `manageStatus` field set to `false`:
 - version: v1
   group: api.example.com
   kind: Foo
-  role: /opt/ansible/roles/Foo
+  role: Foo
   manageStatus: false
 ----
 
-. Then, use the `k8s_status` Ansible module to update the subresource. For
-example, to update with key `foo` and value `bar`, `k8s_status` can be used as
-shown:
-+
+. Then, use the `operator_sdk.util.k8s_status` Ansible module to update the
+subresource. For example, to update with key `foo` and value `bar`,
+`operator_sdk.util` can be used as shown:
 [source,yaml]
```

Comment:
Add +s between the steps and code blocks to make sure they all line up properly when built.